exp/static/exp/img/preview.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```

Comment:
What is this file?

Reply:
preview.svg, or the w3 file? preview.svg is a background image (like the superuser mode image) for the preview items on the individual responses page (like this: https://github.com/lookit/ember-lookit-frameplayer/issues/62#issuecomment-588403569) This may be silly though?